west.yml

```diff
@@ -34,10 +34,10 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 578dac80f7c3bafb3d0e98c0c31b072139251395
+      revision: master
```

Comment:
Those changes should not be part of this PR.

Reply:
Maybe not, then I am abit lost where I should push such canges. This is the manifest that describes which repos and commits this code needs. For me it seems like correct place to put this information.